ui/partials/perf/comparechooserctrl.html

```diff
@@ -27,21 +27,23 @@
         <div class="panel-heading">New</div>
         <div class="panel-body">
           <label for="new-project-selector">Project</label>
-          <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects" ng-change="updateNewRevisionTips()"/>
+          <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects" ng-change="updateNewRevisionTips();getPreviousRevision()"/>
           <label for="new-revision-input">Revision</label>
             <div class="form-group" ng-class="{'has-error': newRevisionError}">
-            <input id="new-revision-input" maxlength="12" class="form-control" type="text" ng-model="newRevision" placeholder="Select or enter a revision"/>
+            <input id="new-revision-input" maxlength="12" class="form-control" type="text" ng-model="newRevision" placeholder="Select or enter a revision" ng-change="getPreviousRevision()"/>
```

Comment:
I think we have getPreviousRevision() in too many places, it should not be populating the right "new" revision field.  I could be mixing this up slightly though.